source/threads.cpp

```diff
@@ -8,7 +8,7 @@ Result Threads::create(ThreadFunc func) {
   Thread thread;
 
   Result res;
-  if (R_FAILED( res = threadCreate(&thread, func, nullptr, 0x2000, 0x2B, -2)))
+  if (R_FAILED( res = threadCreate(&thread, func, nullptr, NULL, 0x2000, 0x2B, -2)))
```

Comment:
Should be nullptr instead of NULL like the parameter before

Reply:
```suggestion
  if (R_FAILED( res = threadCreate(&thread, func, nullptr, nullptr, 0x2000, 0x2B, -2)))
```